### PR TITLE
Fix for #78 check-smart.rb ignoring 'override'

### PR DESCRIPTION
**Pull request by sovaa**
## Pull Request Checklist

**Is this in reference to an existing issue?**

#### General

- [ ] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)

- [ ] Update README with any necessary configuration snippets

- [ ] Binstubs are created if needed

- [ ] RuboCop passes

- [ ] Existing tests pass

#### New Plugins

- [ ] Tests

- [ ] Add the plugin to the README

- [ ] Does it have a complete header as outlined [here](http://sensu-plugins.io/docs/developer_guidelines.html#coding-style)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
+## [2.4.1] - 2017-08-07
```
Comment (majormoses) [inline CHANGELOG.md]: remove the versioning, it is best to leave this to the maintainer as there no guarantee on merge order or release dates.

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
...
+### Fixed
```
Comment (majormoses) [inline CHANGELOG.md]: Move this under `### [Unreleased]`

**Comment by sovaa** [comment]
With fix:

```
[root@host ~]# grep -nA14 "def check_smart_capability" check-smart.rb 
79:  def check_smart_capability!
80-    output = `sudo #{@smart_binary} -i #{device_path}`
81-    @smart_available = !output.scan(/SMART support is:\s+Available/).empty?
82-    @smart_enabled = !output.scan(/SMART support is:\s+Enabled/).empty?
83-    @capability_output = output
84-  end
85-
86-  # Check the SMART health
87-  #
88-  def check_health!
89-    output = `sudo #{@smart_binary} -H #{device_path}`
90-    @smart_healthy = !output.scan(/PASSED|OK$/).empty?
91-    @health_output = output
92-  end
93-end
[root@host ~]# grep override smart.json 
	  { "path": "sda", "override": "/dev/sda -d sat+megaraid,0" }
[root@host ~]# /opt/sensu/embedded/bin/ruby ./check-smart.rb -j smart.json
CheckSMART OK: PASSED
```

Without fix:

```
[root@host ~]# grep -nA14 "def check_smart_capability" check-smart.rb 
79:  def check_smart_capability!
80-    output = `sudo #{@smart_binary} -i #{@device_path}`
81-    @smart_available = !output.scan(/SMART support is:\s+Available/).empty?
82-    @smart_enabled = !output.scan(/SMART support is:\s+Enabled/).empty?
83-    @capability_output = output
84-  end
85-
86-  # Check the SMART health
87-  #
88-  def check_health!
89-    output = `sudo #{@smart_binary} -H #{@device_path}`
90-    @smart_healthy = !output.scan(/PASSED|OK$/).empty?
91-    @health_output = output
92-  end
93-end
[root@host ~]# grep override smart.json
	  { "path": "sda", "override": "/dev/sda -d sat+megaraid,0" }
[root@host ~]# /opt/sensu/embedded/bin/ruby ./check-smart.rb -j smart.json
CheckSMART UNKNOWN: smartctl 6.2 2013-07-26 r3841 [x86_64-linux-3.10.0-514.6.2.el7.x86_64] (local build)
Copyright (C) 2002-13, Bruce Allen, Christian Franke, www.smartmontools.org

Smartctl open device: /dev/sda failed: DELL or MegaRaid controller, please try adding '-d megaraid,N'
```

**Comment by majormoses** [comment]
released: https://rubygems.org/gems/sensu-plugins-disk-checks/versions/2.4.1

**Pull request merged**